arch/arm/include/irq.h

```diff
@@ -66,8 +66,6 @@
  * Pre-processor Prototypes
  ****************************************************************************/
 
-#define up_getsp() (uintptr_t)__builtin_frame_address(0)
```

Comment:
I think this was added specially for tlsr82. @anchao @xiaoxiang781216 please confirm.

Reply:
no, tlsr82 can pass the internal ci without error.